lib/mp-readline/readline.c

```diff
@@ -98,6 +99,38 @@ typedef struct _readline_t {
 
 STATIC readline_t rl;
 
+size_t get_previous_word_cursor_pos() {
+  size_t cursor_pos = rl.cursor_pos;
+  bool in_leading_nonalphanum = true;
+  while (cursor_pos > 0) {
+    if (!isalnum((unsigned char) rl.line->buf[cursor_pos - 1])) {
```

Comment:
Ah and indent should be 4 tabs, see surrounding code

Reply:
Fixed.